script/load-pg-dump

```diff
@@ -65,6 +68,12 @@ while getopts "hcd:u:" opt; do
 done
 shift "$((OPTIND-1))" # Shift off the options and optional
 
+pg_host=
+if [ -n "$host" ]; then
+  pg_host="-h $host"
+fi
```

Comment:
Would using one variable `pg_host` and using `localhost` as default for that be simpler than what we are doing here?

Reply:
it would be easier indeed but it would break the current behavior of the script. So it is up to you project owners and script users to decide what to do here ;)

Reply:
I am not sure I understand how would it break the current behavior. are we talking about PGHOST?

Reply:
Because the default is to use the unix socket instead of localhost